src/components/checkbox/README.md

```diff
@@ -0,0 +1,18 @@
+## Checkbox component
+
+The implementation requires controlled behaviour, so the component requires to be wrapped
+and provided with state and handler.
+
+### Props:
+
+```typescript
+interface CheckboxProps {
+  checked: boolean
+  onChange: (e: React.ChangeEvent<HTMLInputElement>) => void
+  id?: string
+  label?: string
+  labelPosition?: "left" | "right"
+  ref?: React.MutableRefObject<HTMLInputElement>
+  className?: string
+}
```

Comment:
The props should include also the `disabled` property for disabling the control (and the relevant CSS handling of this).

Reply:
I agree with you, I just didn't see the disabled state in Figma, will ask @oyabun

Reply:
Done

Reply:
Nice! Thanks! Only thing to discuss would be if we need to have the disabled state as an explicit story or as a knob control to avoid too many stories. TBD

Reply:
Using knob now